src/qibolab/_core/instruments/qm/program/instructions.py

```diff
@@ -170,9 +171,14 @@ def program(
     args: ExecutionArguments,
     options: ExecutionParameters,
     sweepers: list[ParallelSweepers],
+    offsets: list[tuple[ChannelId, float]],
 ):
     """QUA program implementing the required experiment."""
     with qua.program() as experiment:
+        # force offset setting due to a bug in newer QUA versions and the OPX1000
```

Comment:
Maybe even prepend a `TODO: ` or `FIXME: ` tag.

Reply:
Done in 79f594fb8e19fb8430b4747132f468d8e0580bb7, also for the other issue with `/` described above.